src/zeroD/Reactor.cpp

```diff
@@ -187,7 +187,7 @@ void Reactor::updateConnected(bool updatePressure) {
     m_thermo->saveState(m_state);
 
     // Update the mass flow rate of connected flow devices
-    double time = m_net->time();
+    double time = (m_net!=NULL) ? m_net->time() : t0;
```

Comment:
```suggestion
    double time = (m_net != NULL) ? m_net->time() : 0.0;
```

Cantera's style is to have space around the operators.

Reply:
Also, you should compare to `nullptr` rather than `NULL` to be more in line with C++11.